site/content/guides/using-pw-test-command.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Using the pw-test Command - Checkly Guides
+displayTitle: Using the pw-test Command
+navTitle: Using pw-test
+weight: 15
+beta: false
+description: >-
+  Learn how to use the pw-test command to run Playwright tests with Checkly monitoring. Convert your existing Playwright tests into scheduled monitors with a single command.
+tags:
+  - CLI
+  - Playwright
+---
+
+Run Playwright tests with Checkly monitoring features. Convert to a check suite once you're ready to start monitoring.
+
+## Basic syntax
+
+```bash
+npx checkly pw-test [checkly flags] -- [playwright flags]
+```
+
+Use `--` to separate Checkly flags from Playwright flags.
+
+## What is pw-test?
+
+The `pw-test` command:
+- Runs your existing Playwright tests
+- Records results to Checkly automatically
+- Creates scheduled monitors with `--create-check`
+- Uses your Playwright configuration file options
+
+## Quick examples
+
+### Run tests remotely in Checkly
+
+
+```bash
+# Single browser
+npx checkly pw-test -- --project=chromium
+
+# Multiple browsers
+npx checkly pw-test -- --project=chromium --project=firefox
+
+# Filter by tag
+npx checkly pw-test --location="eu-west-1" --location="us-west-1" -- --grep="@smoke"
+```
+
+By default, `pw-test` uses N.Virginia (`us-east-1`) to run. Use any [global location](/docs/monitoring/global-locations/) or [private location](/docs/private-locations/).
+
+### Create monitors
+
+Add `--create-check` to convert suites of tests into scheduled playwright check suites:
+
+```bash
+# Basic check suite pulling all tests in Chromium every 10 minutes
+npx checkly pw-test --create-check --frequency=10m -- --project=chromium
+
+# Multiple locations pulling all tests tagged critical, and run every 5 minutes
+npx checkly pw-test --create-check --frequency=5m --location=us-east-1 --location=eu-west-1 -- --grep="@critical"
+
+# Mobile tagged tests every 15 mins, in Mobile Chrome and the default location.
+npx checkly pw-test --create-check --frequency=15m -- --project="Mobile Chrome" --grep="@mobile"
+```
+
+## Advanced examples
+
+### Browser and device selection
+
+```bash
+# Desktop browsers
+npx checkly pw-test -- --project="Desktop Chrome" --project="Desktop Firefox"
+
+# Mobile devices
+npx checkly pw-test -- --project="Mobile Safari" --project="Iphone 14"
+```
+
+### Test filtering
+
+```bash
+# All tests tagged smoke
+npx checkly pw-test -- --grep="@smoke"
+
+# Exclude flaky tagged tests
+npx checkly pw-test -- --grep-invert="@flaky"
+
+# Create a check with tests tagged critical that runs every 10 minutes
+npx checkly pw-test --create-check --frequency=10m -- --grep="@critical"
+```
+
+## Configuration
+
+### Artifacts
+
+Configure in your `playwright.config.ts`:
```

Comment:
Does the `playwright.config` win the configuration? If I turn off traces here, we won't have a trace in the test session?

If that's the case, should we make a note?

Reply:
If you turn of traces here, you won't have a trace in checks or in the test session. 

They can override with a custom `testCommand` using `--trace=on`. 
Added this explanation, thanks!